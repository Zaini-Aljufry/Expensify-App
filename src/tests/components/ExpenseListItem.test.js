import {shallow} from 'enzyme'
import React from 'react';
import ExpenseListItem from '../../components/ExpenseListItem'
import expenses from '../fixtures/expenses'
import toJSON from 'enzyme-to-json'//serializer


test('should render ExpenseListItem with expenses correctly', () => {
    const wrapper = shallow(<ExpenseListItem key={expenses[2].id}{...expenses[2]}/>)
    expect(toJSON(wrapper)).toMatchSnapshot();
})