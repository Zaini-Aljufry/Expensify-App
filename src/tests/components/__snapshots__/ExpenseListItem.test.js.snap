// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseListItem with expenses correctly 1`] = `
<div>
  <Link
    to="/edit/3"
  >
    <h3>
      Kopi Peng
    </h3>
  </Link>
  <p>
    1600
     - 
    345600000
  </p>
</div>
`;
